magento-coding-standard/vendor/squizlabs/php_codesniffer/CodeSniffer.conf

```diff
@@ -0,0 +1,5 @@
+<?php
+ $phpCodeSnifferConfig = array (
```

Comment:
Hello, This PR is not related to the hometask. I would advice to remove this folder from the repo as it is not related to Magento project. Especially takin into consideration that the file (magento-coding-standard/vendor/squizlabs/php_codesniffer/CodeSniffer.conf) is located in vendor directory of other project, which should not be changed. Please commit this folder removal, add it in your project's .gitignore, and keep it where it was but not commit. 

I've checked the hometask in master
https://github.com/ahrrhy/gh_8th_magento2/compare/f6d1be843c282be3b3e86151de3ae87a68ed713c...master
No serious objections, everything looks fine. 

One question regarding the file /etc/view.xml. I don't remember any tasks in hometask related to this file. Please specify what was changed there.